assets/blocks/editor-components/toolbar-dropdown/index.js

```diff
@@ -0,0 +1,78 @@
+import {
+	Button,
+	Dropdown,
+	MenuGroup,
+	MenuItem,
+	NavigableMenu,
+} from '@wordpress/components';
+import classnames from 'classnames';
+
+/**
+ * @typedef {Object} DropdownOption
+ *
+ * @property {string} label Option label.
+ * @property {string} value Option value.
+ */
+/**
+ * Dropdown for the editor toolbar.
+ *
+ * @param {Object}           props
+ * @param {DropdownOption[]} props.options        Dropdown options.
+ * @param {string}           [props.optionsLabel] Options label.
+ * @param {Object}           props.icon           Icon for the toolbar.
+ * @param {string}           props.value          Current value.
+ * @param {Function}         props.onChange       Change function.
+ */
+const ToolbarDropdown = ( {
+	options,
+	optionsLabel,
+	icon,
+	value,
+	onChange,
+} ) => {
+	const selectedOption = options.find( ( option ) => value === option.value );
+
+	return (
+		<Dropdown
```

Comment:
Should we change the border to have a block color as the rest of the dropdown menus?

Reply:
Good catch! Fixed here: https://github.com/Automattic/sensei/pull/3904/commits/bbdf53c850e0ca152162e5479524df7ee8a59491